project/Build.scala

```diff
@@ -1091,73 +1091,79 @@ object Build {
+      commonSettings,
+      publishSettings,
+      name := "Scala.js Stubs",
+      libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value//,
+      //previousArtifactSetting
   )
 
   // Scala.js command line interface
   lazy val cli: Project = Project(
       id = "cli",
-      base = file("cli"),
-      settings = (
-          commonSettings ++ publishSettings ++ fatalWarningsSettings
-      ) ++ Seq(
-          name := "Scala.js CLI",
-          libraryDependencies ++= Seq(
-              "com.github.scopt" %% "scopt" % "3.5.0"
-          ),
-
-          previousArtifactSetting,
-          mimaBinaryIssueFilters ++= BinaryIncompatibilities.CLI,
-
-          // assembly options
-          mainClass in assembly := None, // don't want an executable JAR
-          assemblyOption in assembly ~= { _.copy(includeScala = false) },
-          assemblyJarName in assembly :=
-            s"${normalizedName.value}-assembly_${scalaBinaryVersion.value}-${version.value}.jar"
-      )
+      base = file("cli")
+  ).settings(
+      commonSettings,
+      publishSettings,
+      fatalWarningsSettings,
+      name := "Scala.js CLI",
+      libraryDependencies ++= Seq(
+          "com.github.scopt" %% "scopt" % "3.5.0"
+      ),
+
+      previousArtifactSetting,
+      mimaBinaryIssueFilters ++= BinaryIncompatibilities.CLI,
+
+      // assembly options
+      mainClass in assembly := None, // don't want an executable JAR
+      assemblyOption in assembly ~= { _.copy(includeScala = false) },
+      assemblyJarName in assembly :=
+        s"${normalizedName.value}-assembly_${scalaBinaryVersion.value}-${version.value}.jar"
   ).dependsOn(tools)
 
   // Test framework
   lazy val testInterface = Project(
       id = "testInterface",
-      base = file("test-interface"),
-      settings = (
-          commonSettings ++ publishSettings ++ myScalaJSSettings ++ fatalWarningsSettings
-      ) ++ Seq(
-          name := "Scala.js test interface",
-          delambdafySetting,
-          previousArtifactSetting,
-          mimaBinaryIssueFilters ++= BinaryIncompatibilities.TestInterface
-      )
+      base = file("test-interface")
+  ).settings(
+      commonSettings,
+      publishSettings,
+      myScalaJSSettings,
+      fatalWarningsSettings,
+      name := "Scala.js test interface",
+      delambdafySetting,
+      previousArtifactSetting,
+      mimaBinaryIssueFilters ++= BinaryIncompatibilities.TestInterface
   ).withScalaJSCompiler.dependsOn(library)
 
   lazy val jUnitRuntime = Project(
     id = "jUnitRuntime",
-    base = file("junit-runtime"),
-    settings = commonSettings ++ publishSettings ++ myScalaJSSettings ++
-      fatalWarningsSettings ++ Seq(name := "Scala.js JUnit test runtime")
+    base = file("junit-runtime")
+  ).settings(
+    commonSettings,
+    publishSettings,
+    myScalaJSSettings,
+    fatalWarningsSettings,
+    name := "Scala.js JUnit test runtime"
   ).withScalaJSCompiler.dependsOn(testInterface)
```

Comment:
Since you're messing with these lines anyway, could you be a boy scout and fix the indentation inside the `()` to be 4 spaces instead of 2 (inside `Project( ... )` and inside `.settings( ... )`?